solutions/usaco-1091.mdx

```diff
@@ -0,0 +1,14 @@
+---
+id: usaco-1091
```

Comment:
> Couldn't find corresponding problem for internal solution with frontmatter ID usaco-1091. If this problem is no longer in any module, add it to content/extraProblems.json.

*https://vercel.com/cpinitiative/usaco-guide/CLkHRJJStkKpg34HZW6eKF4UzaZM*

I think this is breaking the build